docs/design/debug-api.md

```diff
@@ -0,0 +1,352 @@
+            "gas": "0x857c",
+            "gasUsed": "0x523",
+            "input": "0x70a082310000000000000000000000003ee18b2214aff97000d974cf647e7c347e8fa585",
+            "output": "0x00000000000000000000000000000000000000000000000000001692033dcb57",
+            "calls": [
+              {
+                "type": "DELEGATECALL",
+                "from": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+                "to": "0xa2327a938febf5fec13bacfb16ae10ecbc4cbdcf",
+                "gas": "0x808c",
+                "gasUsed": "0x211",
+                "input": "0x70a082310000000000000000000000003ee18b2214aff97000d974cf647e7c347e8fa585",
+                "output": "0x00000000000000000000000000000000000000000000000000001692033dcb57"
+              }
+            ]
+          },
+          {
+            "type": "CALL",
+            "from": "0x3ee18b2214aff97000d974cf647e7c347e8fa585",
+            "to": "0x98f3c9e6e3face36baad05fe09d375ef1464288b",
+            "value": "0x0",
+            "gas": "0x4f9f",
+            "gasUsed": "0x46c6",
+            "input": "0xb19a437e000000000000000000000000000000000000000000000000000000006eca00000000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000000f000000000000000000000000000000000000000000000000000000000000008501000000000000000000000000000000000000000000000000000000001debea42000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48000267c46aa713cfe47608dd1c16f8a0325208df084c3cbebf9f366ad0eafc2653e4000100000000000000000000000000000000000000000000000000000000001e8542000000000000000000000000000000000000000000000000000000",
+            "output": "0x000000000000000000000000000000000000000000000000000000000001371e",
+            "calls": [
+              {
+                "type": "DELEGATECALL",
+                "from": "0x98f3c9e6e3face36baad05fe09d375ef1464288b",
+                "to": "0x8c0041566e0bc27efe285a9e98d0b4217a46809c",
+                "gas": "0x3b88",
+                "gasUsed": "0x3377",
+                "input": "0xb19a437e000000000000000000000000000000000000000000000000000000006eca00000000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000000f000000000000000000000000000000000000000000000000000000000000008501000000000000000000000000000000000000000000000000000000001debea42000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48000267c46aa713cfe47608dd1c16f8a0325208df084c3cbebf9f366ad0eafc2653e4000100000000000000000000000000000000000000000000000000000000001e8542000000000000000000000000000000000000000000000000000000",
+                "output": "0x000000000000000000000000000000000000000000000000000000000001371e"
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+
+### Method description
+
+`debug_getModifiedAccountsByHash` - Returns all accounts that have changed between the two blocks specified. 
```

Comment:
Please add a high level description of the logic to enable this in terms of other calls. That is what mirror node calls will the logic be making and what data formatting might be necessary.

Reply:
Hi Nana, I have explored the possibility of implementing this method, but to my understanding currently it would not be possible. This method checks if an account has changed from block A to block B, meaning in the past, but currently in the mirror node we cannot check an accounts state in previous blocks.
Same goes to debug_getModifiedAccountsByNumber

Reply:
This is not a priority so should be done after the main debug transaction API is complete. 
In which case just remove this mention from the doc and when we get to it we can always update the design doc.

Reply:
Well, in here https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#debuggetmodifiedaccountsbyhash
It is actually mentioned that a change is  "A change is defined as a difference in nonce, balance, code hash, or storage hash.." , so this complicates things a little bit, if we want to do it in the same way

Reply:
Yes, please remove but capture findings and suggestions in a separate ticket for future consideration.